src/components/AgeRangeField/AgeRangeInput.tsx

```diff
@@ -0,0 +1,343 @@
+import { isEmpty } from 'lodash'
+import React, { CSSProperties, useState } from 'react'
+import { useLocale } from '../../i18n'
+import { ExternalStyles, Theme, colors, focusBoxShadow, useStyles } from '../../styles'
+import { Button } from '../Button'
+import { Dropdown, DropdownItem } from '../Dropdown'
+import { FocusManagerContainer } from '../FocusManagerContainer'
+import { Icon } from '../Icon'
+import { Text } from '../Text'
+import { TextInput } from '../TextField'
+import { Tooltip } from '../Tooltip'
+import { convertUnitAgeRangeEnumToLocaleText } from './converter'
+import { AgeRange, AgeRangeUnitEnum } from './model'
+import { getAvaibleAgeRangeUnits } from './util'
+
+interface Placeholders {
+  start?: string
+  end?: string
+}
+
+export interface AgeRangeInputProps {
```

Comment:
Do you really need to declare all these properties? 

I believe you can work with something like `InputProps` or `React.InputHTMLAttributes<HTMLInputElement>`.